### ISSUE TITLE
add spinner for page loads in web console

### ISSUE THREAD

**Issue by kenirwin** [Miamiohlibs/SoftwareCheckout2#118]
* stats daily summary
* stats summary
* uid load
* log load

can we just automatically add it to every page on the web side? need to avoid adding it to the api router.

**Comment by kenirwin** [comment]
see "spinner" branch for early attempt -- we can get a spinner to load, but it's still there when the user uses the "back" button to go to a previous page. Tried and failed to find a window event (popstate) that would fire when they go back using the back button, but it didn't work. 

next thing to try: if we rebuild the console in react+express, would that work better?

Other things we looked at:
* this would be easier to manage if the ejs page were to do the content loading on the client side instead of express doing it on server side, but that would mean exposing our API credentials in the client side and that's no good.
* could try making the spinner go away after some number of seconds but the page load times vary too much to make that a good option
